dist/tools/ci/build_and_test.sh

```diff
@@ -88,8 +88,7 @@ then
         #   Remove all but `${CI_BASE_BRANCH}` parameters to cppcheck (and remove second
         #   invocation) once all warnings of cppcheck have been taken care of
         #   in ${CI_BASE_BRANCH}.
-        run ./dist/tools/cppcheck/check.sh ${CI_BASE_BRANCH} --diff-filter=MR --error-exitcode=0
-        run ./dist/tools/cppcheck/check.sh ${CI_BASE_BRANCH} --diff-filter=AC
+        run ./dist/tools/cppcheck/check.sh ${CI_BASE_BRANCH} --diff-filter=ACMR
```

Comment:
Looking at the check.sh code [here](https://github.com/RIOT-OS/RIOT/blob/master/dist/tools/cppcheck/check.sh#L35), `ACMR` is the default value.